luke/index.js

```diff
@@ -0,0 +1,115 @@
+// ! 1. Person Class
+class Person {
+  constructor(name, age) {
+    this.name = name;
+    this.age = age;
+  }
+
+  describes() {
+    return `${this.name}, is ${this.age} years old.`;
+  }
+}
+
+const personOne = new Person("John", 19);
+
+console.log(personOne.describes());
+
+//! 2. Volume
+
+class Volume {
+  constructor(radius, height) {
+    this.radius = radius;
+    this.height = height;
+  }
+
+  getVolume() {
+    let volume = Math.PI * (Math.pow(this.radius, 2) * this.height);
+    return volume.toFixed(4);
+  }
+}
+
+const cylinder = new Volume(4, 3);
+
+console.log(cylinder.getVolume());
+
+//! 3. Tick Tock
+class Clock {
+  constructor(object) {
+    this.template = object.template;
+  }
+
+  render = () => {
+    let date = new Date();
+    let hours = date.getHours();
+    if (hours < 10) hours = "0" + hours;
+
+    let mins = date.getMinutes();
+    if (mins < 10) mins = "0" + mins;
+
+    let secs = date.getSeconds();
+    if (secs < 10) secs = "0" + secs;
+
+    let output = this.template
+      .replace("h", hours)
+      .replace("m", mins)
+      .replace("s", secs);
+
+    console.log(output);
+  };
+
+  stop() {
+    clearInterval(this.timer);
+  }
+
+  start() {
+    this.render();
+    this.timer = setInterval(this.render, 1000);
+  }
+}
+
+let clock = new Clock({ template: `h:m:s` });
+clock.stop();
```

Comment:
Task 3 ==> you can stop the timer using setInterval like for example      setInterval(() =>clock.stop(), 3000);
but before this code we need clock.start() then the timer stops after 3 secs
- using your code you get this output 150.7964

Reply:
thanks for the help :)

Reply:
Any time:)